migrations/1730216056247_add-index-to-retrieval-orders.js

```diff
@@ -0,0 +1,11 @@
+exports.shorthands = undefined
+
+exports.up = (pgm) => {
+  pgm.createIndex('retrieval_orders', ['retrieval_collection_id'], {
+    method: 'btree'
```

Comment:
Why did you choose btree

Reply:
In the JIRA ticket I showed how I came up with these. I used an website that analyzes the schema, and it recommended this indexes